tools/runners/run-test-suite-test262.py

```diff
@@ -15,17 +15,27 @@
 # limitations under the License.
 
 from __future__ import print_function
-import sys
 import os
-import subprocess
 import shutil
+import signal
+import subprocess
+import sys
+
 
 def get_platform_cmd_prefix():
     if sys.platform == 'win32':
         return ['cmd', '/S', '/C']
     return ['python2']  # The official test262.py isn't python3 compatible, but has python shebang.
 
 
+
+
+def restore_timezone(original_timezone):
+    if sys.platform == 'win32':
```

Comment:
This check seems redundant to me since this function is only invoked whether `sys.platform == 'win32'` evaluates to true.

Reply:
Thanks for the notice, it is a leftover after a local refactoring. I found a better solution and removed more redundancy.